### DISCUSSION TITLE
add disconnected websocket hiding

### DISCUSSION THREAD

**Discussion by L0ric0** [jcorporation/myMPD#483] [Ideas]
when the websocket disconnects shortly it is sometimes impossible to do anything for the time until the ws reconnects. It would probably help the userexpirience to implement something to hide this. Like queueing up the commands for a certain time and issuing as soon as the ws reconnects and maybe caching on the browser side. should the ws take to long the commands should probably not be issued.

**Comment by jcorporation** [comment]
I am not sure if your proposal is better. myMPD uses the websocket connection only for notifications pushed from the server to clients. No need to queue items. But if the websocket disconnects there must be a problem and deactivating the gui temporarily seems the best solution for me.
